includes/discovery/toner.inc.php

```diff
@@ -31,7 +31,7 @@
                     $capacity_oid = ".1.3.6.1.2.1.43.11.1.1.8.1.$index";
                 }
 
-                $descr = trim(str_replace("\n", '', str_replace('"', '', snmp_get($device, $descr_oid, '-Oqv'))));
+                $descr = trim(str_replace("\n", '', preg_replace('/[^ \w]+/', '', snmp_get($device, $descr_oid, '-Oqv'))));
```

Comment:
I don't know what it is we are trying to fix so I'm not certain but the new preg_replace is check if we have a space before anything else - is that how it is?